### DISCUSSION TITLE
Welcome to poly-flashloan-bot Discussions!

### DISCUSSION THREAD

**Discussion by yuichiroaoki** [yuichiroaoki/poly-flashloan-bot#30] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Peacelight6** [comment]
Hi, 

When i run the 'yarn install' module there is happening this error: 
![Polyflash error](https://user-images.githubusercontent.com/102015640/165081551-25887a5d-fd6a-4614-b5b5-d6e2a157b074.jpg)
